### PR TITLE
virtio_fs: add queue_size test case

### PR DESCRIPTION
**Pull request by xiagao**
IO test on shared dir in different queue_size situation.
ID: 1569

### CODE REVIEW

**Comment by fbq815** [comment]
Test result:
RHEL8:
RESULTS    : PASS 4 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
RHEL9:
RESULTS    : PASS 4 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0

LGTM

**Comment by zhenyzha** [comment]
3 cases pass on arm platform

LGTM.Acked-by: Zhenyu Zhang <zhenyzha@redhat.com>

**Review by zhenyzha — APPROVED** [review]

**Comment by xiagao** [comment]
@YongxueHong  Could you pls help review this patch?

File: qemu/tests/cfg/virtio_fs_queue_size.cfg [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    remove_fs_source = yes
+    fs_target = 'myfs'
+    share_mem = yes
+    test_file = 'test_file'
+    fs_dest = '/mnt/${fs_target}'
+    vm_mem_share = yes
+    vm_mem_backend = memory-backend-file
+    vm_mem_backend_path = /dev/shm
+    driver_name = viofs
+    fs_binary_extra_options = ''
```
Comment (YongxueHong) [inline qemu/tests/cfg/virtio_fs_queue_size.cfg:26]:
```suggestion
```
Looks like we don't have to assign an empty string here.
Maybe you have other concerns? Please let me know. Thanks.

Comment (xiagao) [reply qemu/tests/cfg/virtio_fs_queue_size.cfg]: You're right, deleted it already. Thanks.

**Review by YongxueHong — APPROVED** [review]

**Comment by YongxueHong** [comment]
> Test result: RHEL8: RESULTS : PASS 4 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0 RHEL9: RESULTS : PASS 4 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
> 
> LGTM

Hi @fbq815 
Just a reminder that suggest clicking the `approve` button to approve the PR.

**Pull request merged**